apps/src/templates/courseOverview/CourseEditor.js

```diff
@@ -98,14 +112,30 @@ export default class CourseEditor extends Component {
             ))}
           </select>
         </label>
+        <VisibleInTeacherDashboard
+          paramName="visible"
+          checked={this.state.visible}
+          disabled={!!this.state.pilotExperiment}
+          onChange={e => this.setState({visible: e.target.checked})}
+        />
         <label>
-          Pilot Experiment
+          Can be recommended (aka stable)
           <input
-            name="pilot_experiment"
-            defaultValue={pilotExperiment}
-            style={styles.input}
+            name="is_stable"
+            type="checkbox"
+            defaultChecked={this.props.isStable}
+            style={styles.checkbox}
           />
+          <p>
+            If checked, this unit will be eligible to be the recommended version
+            of the unit. The most recent eligible version will be the
+            recommended version.
+          </p>
         </label>
+        <PilotExperiment
+          value={this.state.pilotExperiment}
+          onChange={e => this.setState({pilotExperiment: e.target.value})}
```

Comment:
What happens if visible was set and then you start typing in a pilot experiment?

Reply:
The same thing happens - the checkbox gets greyed out, and then it's set as not visible. The behavior should be the same as what's on the script edit UI.

Reply:
Ah ok I see it now.